defaults.yaml

```diff
@@ -11,6 +11,7 @@ conf:
   minion_2_name: k8s-minion-2
   minion_3_name: k8s-minion-3
   kubernetes_nodes_base_box_id: "bento/centos-7.4"
+  verbose_output: "off"
```

Comment:
Idea: we could rename this to `additional_ansible_arguments` or something like that. So we can get rid of the `if` in the shell script, and becomes a responsibility of the user to set the right parameters.

Way more flexibility and less complexity on our side. WDYT?